### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #989 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#458]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 7.63 KB...]
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.12 sec - in smartcity.accessibility.socialnetwork.ReviewCommentTest
Running smartcity.accessibility.socialnetwork.HelpfulnessTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.003 sec - in smartcity.accessibility.socialnetwork.HelpfulnessTest
Running smartcity.accessibility.socialnetwork.UserFunctionalityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.082 sec - in smartcity.accessibility.socialnetwork.UserFunctionalityTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.015 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.UserPinTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.093 sec - in smartcity.accessibility.socialnetwork.UserPinTest
Running smartcity.accessibility.socialnetwork.DeleteReviewsTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.141 sec - in smartcity.accessibility.socialnetwork.DeleteReviewsTest
Running smartcity.accessibility.socialnetwork.ScoreTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.785 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.database.UserManagerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 7.932 sec - in smartcity.accessibility.database.UserManagerTest
Running smartcity.accessibility.database.LocationManagerTest
Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.963 sec - in smartcity.accessibility.database.LocationManagerTest
Running smartcity.accessibility.database.ReviewManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.519 sec - in smartcity.accessibility.database.ReviewManagerTest
Running smartcity.accessibility.database.UserProfileManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.238 sec - in smartcity.accessibility.database.UserProfileManagerTest
Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.378 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  ProfileImgServiceTest.getProfileImageDefault:75 array lengths differed, expected.length=5552 actual.length=2262

Tests run: 83, Failures: 1, Errors: 0, Skipped: 3

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:25 min
[INFO] Finished at: 2017-06-24T18:27:17+00:00
[INFO] Final Memory: 34M/217M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] c774dfcea094b9408fac4050bc3fcf8bcbceead9 - fix profile users is not created

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/989/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
